hooks/useAvalancheCenterMetadata.ts

```diff
@@ -77,7 +77,9 @@ const fetchAvalancheCenterMetadata = async (nationalAvalancheCenterHost: string,
         url,
       },
     });
-    throw parseResult.error;
+    // If we throw here, we'll just trigger retries forever based on react-query's retry policy.
+    // Since we know we're getting an unparseable result, we should stop asking.
+    return null;
```

Comment:
Doing this in one hook - but no others - is going to surprise someone who is not expecting that. Perhaps update the schema if we're simply out-of-date or use a deny-list of nonconforming centers in the list?

Reply:
The right solution is not immediately obvious to me. Centers are conforming one day and non-conforming the next (MSAC is the current offender) so I think maintaining a deny list in code will be toilsome. A cloud-based approach might work but I don't have time for bigger solutions this week...

I observed infinite retry in the logs, otherwise I wouldn't have bothered making this change.

Turning this into an issue so it can get resolved more solidly.